tests-fuzz/targets/fuzz_create_table.rs

```diff
@@ -61,16 +61,31 @@ impl Arbitrary<'_> for FuzzInput {
 
 fn generate_expr(input: FuzzInput) -> Result<CreateTableExpr> {
     let mut rng = ChaChaRng::seed_from_u64(input.seed);
-    let create_table_generator = CreateTableExprGeneratorBuilder::default()
-        .name_generator(Box::new(MappedGenerator::new(
-            WordGenerator,
-            merge_two_word_map_fn(random_capitalize_map, uppercase_and_keyword_backtick_map),
-        )))
-        .columns(input.columns)
-        .engine("mito")
-        .build()
-        .unwrap();
-    create_table_generator.generate(&mut rng)
+    let metric_engine = rng.gen_bool(0.5);
```

Comment:
@WenyXu do we need some adjust after #3614 merged?

I can see we already have test coverage here for "create metric table".

Reply:
> @WenyXu do we need some adjust after #3614 merged?
> 
> I can see we already have test coverage here for "create metric table".

Nope, all tests passed